### ISSUE TITLE
Move beta label underneath bottom sheet title for long titles

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#16513]
When the title label is longer than one line the beta label should be shown underneath the title (like for a11y sizes).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7449)

**Comment by data-sync-user** [comment]
➤ Alina Moldovan commented:

Verified as fixed using main  and iPhone 15 Pro Simulator. 

Was not able to test this using the real device and the nightly build because the end user has no possibility to change the name of the drawer. 

!image-20231027-135232.png|width=475,height=1015!

!image-20231027-135306.png|width=468,height=1009!
